frontend/src/pages/projects/screens/detail/storage/StorageList.tsx

```diff
@@ -69,15 +68,15 @@ const StorageList: React.FC = () => {
         }
       >
         {!isPvcsEmpty ? (
-          <StorageTable pvcs={pvcs} refresh={refresh} onAddPVC={() => setOpen(true)} />
+          <StorageTable pvcs={pvcs} refresh={refreshPvcs} onAddPVC={() => setOpen(true)} />
         ) : null}
       </DetailsSection>
       {isOpen ? (
         <ClusterStorageModal
           onClose={(submitted) => {
             setOpen(false);
             if (submitted) {
-              refresh();
+              refreshPvcs();
```

Comment:
We might still need to refresh notebooks here, otherwise the connected workbench field won't update on the table